src/main/java/frc/trigon/robot/subsystems/collector/CollectorConstants.java

```diff
@@ -0,0 +1,33 @@
+package frc.trigon.robot.subsystems.collector;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+public class CollectorConstants {
+    private static final int MOTOR_ID = 1;
+    private static final NeutralMode NEUTRAL_MODE_VALUE = NeutralMode.Brake;
+    private static final boolean INVERTED = true;
+    private static final int VOLTAGE_COMP_SATURATION = 11;
+    private static final boolean ENABLE_VOLTAGE_COMPENSATION = true;
```

Comment:
This is not really needed in this case, since configuring the voltage comp is like saying you want to enable it